README.md

```diff
@@ -1,35 +1,35 @@
 # Hello World
 ## 1.Try to solve "Helloworld" on codecheck
-You are in an Open Challenge as tutorial.
-You can learn how to use codecheck through taking this tutorial challenge.
+This is an Open Challenge as well as a tutorial.
+You can learn how to use codecheck by taking this tutorial.
 
-### 1-1.  Make sure "Deadline" and "Time Limit"
-After opening a challenge, you can see the detail of the challenge like this.  
-And you can see that the Deadline or Time limit is displayed on this page.  
-Please keep in mind that you can not submit your answer after the time is over.
+### 1-1.  Keep "Deadline" and "Time Limit" in mind.
+After openinig a challenge, it is possible to see the detail page of a challenge.
+On this detail page you can see the Deadline or Time limit.
+Please keep in mind that you can not submit your answer after the time expired.
 
-### 1-2. Choose the way to solve (Web editor or GitHub)
-You can solve an open challenge in our dedicated Web editor or GitHub.  
+### 1-2. Choose a way to solve (Web Editor or GitHub)
+You can solve an open challenge in our dedicated Web Editor or in GitHub.  
 First of all, **let's click "Open Web Editor" button and open it.**
 
-## 2. Solve Challenge in Web Editor
-- Please proceed to the next step from the Web editor.
+## 2. Solve Challenge in the Web Editor
```

Comment:
**Solve in Web Editor** 

- this is a better simpler title because later in the GitHub section you called it "Solve in GitHub"